applications/asset_tracker/src/env_sensors/CMakeLists.txt

```diff
@@ -5,4 +5,31 @@
 #
 
 zephyr_include_directories(.)
-target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/env_sensors.c)
+
+if (CONFIG_USE_BME680_BSEC)
+	if(CONFIG_FP_HARDABI)
+		target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/bsec.c)
+		set (bsec_dir ${NRF_DIR}/ext/BSEC_1.4.7.4_Generic_Release)
+		set(BSEC_LIB_DIR     ${bsec_dir}/algo/Normal_version/bin/gcc/Cortex_M4F)
+		set(BSEC_INCLUDE_DIR ${bsec_dir}/algo/Normal_version/bin/gcc/Cortex_M4F)
+		if(NOT EXISTS "${BSEC_LIB_DIR}/libalgobsec.a")
+			assert(0 "Could not find BSEC library")
+		endif()
+		target_include_directories(app PRIVATE ${bsec_dir}/API)
+		target_include_directories(app PRIVATE ${bsec_dir}/examples)
+		target_include_directories(app PRIVATE BSEC_INCLUDE_DIR)
```

Comment:
Don't you need to de-reference BSEC_INCLUDE_DIR ?

${BSEC_INCLUDE_DIR}

Reply:
fixed